docs/user-guide/liveness/http-liveness.yaml

```diff
@@ -13,6 +13,8 @@ spec:
       httpGet:
         path: /healthz
         port: 8080
+        httpHeaders:
+          - ['X-Custom-Header', 'Awesome']
```

Comment:
I didn't realize this was valid YAML for initializing a struct - is it?  Can you please use the more conventional syntax:

```
httpHeaders:
  - name: X-Custom-Header
    value: Awesome
```